test/test_display.jl

```diff
@@ -19,6 +19,8 @@ import ArchGDAL; const AG = ArchGDAL
              Field 0 (FID): [OFTReal], 2.0, 3.0, 0.0, 3.0
              Field 1 (pointname): [OFTString], point-a, point-b, a, b
         """
+        @test sprint(print, AG.nextnamedtuple(layer), context=:compact => true) == 
+        "(FID = 2.0, pointname = \"point-a\",  = Geometry: wkbPoint)"
```

Comment:
Thought this looks funny, but that's an empty geometry name of course.

Reply:
Yeah, I was suprised a `Symbol("")` was actually allowed as key in the beginning.